### PR TITLE
LS-7768: Add model no. to the related assets information in asset det…

### PR DESCRIPTION
**Pull request by SmritiLogistimo**
LS-7768: Add model no. to the related assets information in asset details and asset listing api

### CODE REVIEW

File: app/com/logistimo/db/AssetMapping.java [hunk 1]
```diff
@@ -124,6 +124,16 @@ public static AssetMapping findMonitoredAssetMapping(Device monitoringAsset) {
         .getSingleResult();
   }
 
+  public static List<AssetMapping> findMonitoringAssetRelation(Device monitoringAsset) {
```
Comment (kgaurav1) [inline app/com/logistimo/db/AssetMapping.java]: This method can be merged with previous one as only id in query is different.
relevant id should be sent from calling env.

Comment (SmritiLogistimo) [reply app/com/logistimo/db/AssetMapping.java]: In the previous query, we are returning only `getSingleResult()`. But in this case, we need `getResultList()`. 
The above query is used at multiple places, so we need to change the implementations everywhere. That's why added a new query.

File: app/com/logistimo/services/DeviceService.java [hunk 2]
```diff
@@ -2189,27 +2193,41 @@ private void createAssetMapping(AssetMapModel assetMapModel, Device device,
   }
 
   private Map<Integer, AssetMapModel> toAssetMapModels(List<AssetMapping> assetMappingList) {
+    return toAssetMapModels(assetMappingList, false);
+  }
+
+  public Map<Integer, AssetMapModel> toAssetMapModels(List<AssetMapping> assetMappingList, boolean isMonitoringAsset) {
```
Comment (kgaurav1) [inline app/com/logistimo/services/DeviceService.java]: Make this protected from public

File: app/com/logistimo/services/DeviceService.java [hunk 3]
```diff
@@ -1903,7 +1907,7 @@ private void updateTemperatureSensorStatus(Device device, List<String> actSns) {
     }
   }
 
-  private List<DeviceMetaData> getDeviceMetaDatas(Device device) {
+  public List<DeviceMetaData> getDeviceMetaDatas(Device device) {
```
Comment (kgaurav1) [inline app/com/logistimo/services/DeviceService.java]: Make this protected from public

**Pull request merged**